cvmfs/cvmfs_config

```diff
@@ -1231,9 +1235,13 @@ add_user_to_group_fuse() {
 check_auto_mounter() {
   case $sys_arch in
     Linux )
-      $service autofs status > /dev/null 2>&1
-      return $? ;;
-
+        if [[ $service == *systemctl ]]; then
```

Comment:
Is the compatibility symlink from `service` to `systemctl` missing on archlinux?  If it's there, I'd prefer for the time being to stay with it.  In the longer run, I think this script should have wrapper functions for start/stop/reload services that cover both the binary name of the system manager and the correct parameter syntax.